requirements.txt

```
@@ -20,6 +20,7 @@ pydub
 pyyaml>=5.0,<7.0
 semantic_version~=2.0
 typing_extensions~=4.0
+urllib3  # urllib3 is used for Lite support. Version spec can be omitted because urllib3==2.1.0 is prebuilt for Pyodide and urllib>=2.2.0 supports Pyodide as well.
```

I would suggest pinning to major so that if urllib3 comes out with a breaking change (unlikely I know, but still), gradio doesn't break
```suggestion
urllib3~=2.0  # urllib3 is used for Lite support. Version spec can be omitted because urllib3==2.1.0 is prebuilt for Pyodide and urllib>=2.2.0 supports Pyodide as well.
```

---

Right! Thank you :D